lib/Cake/Network/Email/CakeEmail.php

```diff
@@ -1223,7 +1235,7 @@ public static function deliver($to = null, $subject = null, $message = null, $tr
  */
 	protected function _applyConfig($config) {
 		if (is_string($config)) {
-			if (!class_exists($this->_configClass) && !config('email')) {
+			if (!class_exists($this->_configClass) && !$this->_configInstance instanceof $this->_configClass && !config('email')) {
 				throw new ConfigureException(__d('cake_dev', '%s not found.', APP . 'Config' . DS . 'email.php'));
 			}
 			$configs = new $this->_configClass();
```

Comment:
Config instance is still being recreated here which is what was to be avoided :)